__tests__/app.test.js

```diff
@@ -404,4 +404,19 @@ describe("All endpoints", () => {
 				})
 		})
 	})
+
+	//-----#12 POST /api/comments/:comment_id endpoint ----------
+	describe.only("DELETE - /api/comments/:comment_id", () => {
+		test("Status 204 responds with empty body ", () => {
+			return request(app).delete("/api/comments/1").expect(204)
+		})
+		test("400 response with an error message for invalid comment_id", () => {
+			return request(app)
+				.delete("/api/comments/an-invalid-id")
+				.expect(400)
+				.then(({ body: { msg } }) => {
+					expect(msg).toBe("Bad request")
+				})
+		})
+	})
```

Comment:
could do with one more test I think, what happens if the comment your trying to delete is a valid id but isn't in the db?